website/public/site.webmanifest

```diff
@@ -1,33 +1,19 @@
 {
   "name": "Marquee",
   "short_name": "Marquee",
-  "start_url": "/",
-  "display": "standalone",
-  "orientation": "portrait",
-  "theme_color": "#ffffff",
-  "background_color": "#ffffff",
   "icons": [
     {
-      "src": "/favicon/android-chrome-192x192.png",
```

Comment:
🤔  not sure how relative vs absolute is treated here but I would keep the beginning `/`